specs/DesignDiscussions/DynamicTypeAhead.md

```diff
@@ -0,0 +1,281 @@
+# Dynamic Type Ahead Search in Adaptive cards
+
+## Summary
+
+Input choice sets are a heavily used component in adaptive cards and lets developers create short form like experiences to take information from the end user and pass it on to the bot. Dynamic type ahead feature is useful when the user has to select from a large number of choices, a searchable select menu that dynamically loads data as the user searches will help in avoiding the poor experience that currently exists today (a static list of large number of choices). This feature will ultimately expand the horizon of the kind of use cases adaptive cards can be used by developers as large datasets (example addresses, product catalogs, contacts) can be used to take input from users.
```

Comment:
nit: Break "Dynamic type ahead feature is useful when the user has to select from a large number of choices, a searchable select menu that dynamically loads data as the user searches will help in avoiding the poor experience that currently exists today (a static list of large number of choices)." into two sentences for readability.

Change to: "Dynamic type ahead feature is useful when the user has to select from a large number of choices. A searchable select menu that dynamically loads data as the user searches will help in avoiding the poor experience that currently exists today (a static list of large number of choices).